examples/tensorflow-imagenet-resnet50-fgm/demo.ipynb

```diff
@@ -156,6 +156,9 @@
     "# Override the MLFLOW_TRACKING_URI variable, used to connect to MLFlow Tracking service\n",
     "os.environ[\"MLFLOW_TRACKING_URI\"] = MLFLOW_TRACKING_URI\n",
     "\n",
+    "# Path to custom task plugins archives\n",
+    "CUSTOM_PLUGINS_EVALUATION_TAR_GZ = Path(\"custom-plugins-evaluation.tar.gz\")\n",
```

Comment:
If you change the name of the generated tar file in the Makefile, it will need to be updated here to match.